components/AdminUserDashboard.tsx

```diff
@@ -13,6 +13,12 @@ export default function AdminUserDashboard(): JSX.Element {
     push,
     query: { dashboard }
   } = useRouter()
+  const { API_MANAGER_ENABLED } = process.env
```

Comment:
Please update `env.tmp` to mention about this new option.

Reply:
Done in 3a29c33ffe0b1276cc11e44d7ce6ffe766f99d71